src/components/ActionPanel.tsx

```diff
@@ -64,33 +74,46 @@ export default class ActionPanel extends React.Component<Props> {
         const { store, paused } = this.props;
         const { learned, flags } = store;
 
+        const staminaProp = store.resources.get(resources.STAMINA);
+        const stamina = staminaProp ? staminaProp.currentValue : 0;
+
         // TODO: make actionbutton also ignore click when paused
         return (
             <LabeledPanel title="Actions" id="actions">
                 <div>
                     <ActionButton
                         label="Wander"
-                        cost="-5 STA"
+                        cost={`${resources.WANDER_STA} STA`}
                         onClick={this.wander}
                         paused={paused}
+                        locked={stamina <= 0}
```

Comment:
This should be locked if STA is under `WANDER_STA`, not 0

Reply:
oops fixed lol  74ef89b